test/autorest/generated/lrogroup/lroretrys.go

```diff
@@ -93,20 +90,11 @@ func (client *lroRetrysOperations) delete202Retry200CreateRequest() (*azcore.Req
 }
 
 // delete202Retry200HandleResponse handles the Delete202Retry200 response.
-func (client *lroRetrysOperations) delete202Retry200HandleResponse(resp *azcore.Response) (*LroRetrysDelete202Retry200Response, error) {
-	result := LroRetrysDelete202Retry200Response{RawResponse: resp.Response}
-	if val := resp.Header.Get("Location"); val != "" {
-		result.Location = &val
-	}
-	if val := resp.Header.Get("Retry-After"); val != "" {
-		retryAfter32, err := strconv.ParseInt(val, 10, 32)
-		retryAfter := int32(retryAfter32)
-		if err != nil {
-			return nil, err
-		}
-		result.RetryAfter = &retryAfter
+func (client *lroRetrysOperations) delete202Retry200HandleResponse(resp *azcore.Response) (*HTTPResponse, error) {
+	if !resp.HasStatusCode(pollingCodes[:]...) {
```

Comment:
This should be the list of status codes defined by the swagger.

Reply:
Done